reference/7.2/Microsoft.PowerShell.Core/About/about_ANSI_Terminals.md

```diff
@@ -71,43 +70,41 @@ The following members control how or when ANSI formatting is used:
 - `$PSStyle.OutputRendering` is a
   `System.Management.Automation.OutputRendering` enum with the values:
 
-  - **ANSI**: This is the default behavior. ANSI is always passed through
-    as-is.
-  - **PlainText**: ANSI escape sequences are always stripped so that it is only
+  - `ANSI`: ANSI is always passed through as-is.
+  - `PlainText`: ANSI escape sequences are always stripped so that it's only
     plain text.
-  - **Host**: The ANSI escape sequences are removed in redirected or piped
-
-  > [!NOTE]
-  > `$PSStyle.OutputRendering` only applies to rendering in the Host,
-  > `Out-File`, and `Out-String`. The `Out-File` and `Out-String` cmdlets can
-  > alter pure string input based on value of **OutputRendering**.
+  - `Host`: This is the default behavior. The ANSI escape sequences are removed
+    from redirected or piped output. For more information, see
+    [Redirecting output][14].
```

Comment:
Nit: Should the numbering for the link references be reworked so this is `3` instead of `14`?

Reply:
Verbally agreed to leave the numbering as-is.